frontend/src/components/Calendar/Points.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styles from './Points.module.css';
+
+const Points = () => {
+  const scores = [
+    { id: 1, title: 'Study React', points: 10 },
+    { id: 2, title: 'Finish assignment', points: 20 },
+    { id: 3, title: 'Attend team meeting', points: 15  },
+    { id: 4, title: 'Prepare presentation', points: 30 },
+    { id: 5, title: 'Exercise for 30 minutes', points: 25 },
+  ];
+
+  return (
+    <div className={styles.tasks}>
+      <h3 className={styles.button1}>Today's tasks</h3>
```

Comment:
- What will this button lead to? Is this a function that backend should concern about?

Reply:
I think we can consider how to display today's tasks